.github/workflows/release.yml

```diff
@@ -17,10 +17,16 @@ jobs:
         uses: actions/setup-go@v2
         with:
           go-version: 1.16
+      - name: Login to GitHub Packages Docker Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
       - name: Run GoReleaser
         uses: goreleaser/goreleaser-action@v2
         with:
           version: latest
           args: release --rm-dist
         env:
-          GITHUB_TOKEN: ${{ secrets.GORELEASER_TOKEN }}
```

Comment:
Please revert back this line, `GORELEASER_TOKEN` should be used.

Reply:
We don't need a GORELEASER_TOKEN, GITHUB_TOKEN is injected automatically and has the rights.

Reply:
`GITHUB_TOKEN`, as you mentioned, is automatically injected but `secrets.GORELEASER_TOKEN` is a custom variable we are using for the release process here.

Reply:
Is it still necessary ? GITHUB_TOKEN has the right to push packages on the repo and that's all we need.

Reply:
Yes it is :)

Reply:
Alright ;)